docker/inference/dockerfile.torch

```diff
@@ -53,7 +53,7 @@ RUN pip install cupy-cuda115 nvidia-pyindex pybind11 pytest protobuf transformer
 RUN pip install betterproto cachetools graphviz nvtx scipy sklearn
 RUN pip install numba --no-deps
 RUN pip install tritonclient[all] grpcio-channelz
-RUN pip install dask==2021.11.2 distributed==2021.11.2 dask[dataframe]==2021.11.2 dask-cuda
+RUN pip install dask==2021.11.2 distributed==2021.11.2 dask[dataframe]==2021.11.2 dask-cuda22.2.0
```

Comment:
@albert17 is this right? looks different from the other files. Maybe should be `dask-cuda==22.2.0`

Reply:
Oh that was a typo. My bad. Still this file is not needed for rebuild, so no problem for now. But I will update in another Pr tomorrow morning